libs/remix-debug/test/sourceLocationTracker.js

```diff
@@ -63,12 +63,12 @@ tape('SourceLocationTracker', function (t) {
         st.equal(map.start, 35)
 
         map = await sourceLocationTracker.getSourceLocationFromVMTraceIndex('0x0d3a18d64dfe4f927832ab58d6451cecc4e517c5', 45, output.contracts)
-        st.equal(map.file, -1)
+        st.equal(map.file, 1)
 
         map = await sourceLocationTracker.getValidSourceLocationFromVMTraceIndex('0x0d3a18d64dfe4f927832ab58d6451cecc4e517c5', 45, output.contracts)
-        st.equal(map.file, 0)
-        st.equal(map.start, 303)
-        st.equal(map.length, 448)
+        st.equal(map.file, 1)
```

Comment:
there's more than one file in `output.contracts` ?

Reply:
no